data.schema.json

```diff
@@ -46,7 +46,16 @@
           }
         },
         "additionalProperties": false,
-        "required": ["ttl", "kml"]
+        "required": ["ttl", "kml", "md"],
+        "if": {
+          "properties": {
+            "bkg": { "const": true }

```

Comment:
Lehet, hogy úgy is működne, hogy bkg = false és nem-required; else helyett then-el